lib/plop-pack/src/helpers/title.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+const title = function(options) {
+  const t = options.fn(this);
+  return `${t}\n${t.replace(/./g, '=')}`;
+}
```

Comment:
I can't remember how these things work. Has it been tested?

Reply:
Unit tests added!

Reply:
Nice.